src/simmate/apps/badelf/core/electride_finder.py

```diff
@@ -168,14 +168,14 @@ def get_electride_structure(
                 The number of threads to use to perform the bader/zero-flux
                 calculation.
 
-            check_for_covalency (bool):
+            ignore_low_pseudopotentials (bool):
                 Whether to prevent a structure from being found if covalency is
                 found in the structure. It is highly recommended to keep this as
                 True as there is currently no method implemented to handle
                 covalency.
 
```

Comment:
Same comment here about how "It is highly recommended to keep this as True..." should probably read `False` now.